### ISSUE TITLE
Converting a Notebook to slides with dark themes don't always work well

### ISSUE THREAD

**Issue by baco** [jupyter/help#247]
I am using the conversion commands:

* `jupyter nbconvert mynotebook.ipynb --to slides --SlidesExporter.reveal_theme=black`
* `jupyter nbconvert mynotebook.ipynb --to slides --SlidesExporter.reveal_theme=league`

When converting notebooks to the Reveal.js dark themes sometimes the background doesn't change to dark and some fonts do change to bright colors making them invisible (header fonts in theme black, blood, night, moon), and sometimes the conversion takes the dark background but some fonts remain in black and get confused with the new background (normal fonts in theme league).

In some cases `code` tagged text get both a dark font and a dark background, for instance in theme blood.

The Jupyter input-area never uses some of the dark themes for code provided in Revel.js, for example Monokai.

**Comment by takluyver** [comment]
cc @damianavila

**Comment by damianavila** [comment]
Yes, dark themes need further customization to be able to use them. You can probably add css adjustment to make it look OK, but trying to support dark themes on the default experience is something we probably need to check in terms of cost vs. rewards benefit, because it is probably not trivial to make it right and I am no sure how many are actually using dark themes.
